source/elements/oneDAL/source/_static/style.css

```diff
@@ -0,0 +1,100 @@
+/* override table width restrictions */
```

Comment:
Files in this directory are not included when we build the full spec. It also needs to be in source/_static

Reply:
I think that will drop entire local css part. @outoftardis - my assumption that we have css aligned across other docs and there is no special things related to oneDAL